src/view/plandetail/SavePlanCollageImageDialog.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+    Box,
+    Button,
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    ModalOverlay,
+} from "@chakra-ui/react";
+import { CollageTemplate } from "src/view/plandetail/CollageTemplate";
+
+interface SavePlanCollageImageDialogProps {
+    isOpen: boolean;
+    onClose: () => void;
+}
```

Comment:
Propsの定義は通常こんな感じでやりますー
```suggestion
type Props = {
    isOpen: boolean;
    onClose: () => void;
}
```